src/recipe.rs

```diff
@@ -103,6 +103,10 @@ impl<'src, D> Recipe<'src, D> {
     !self.attributes.contains(&Attribute::NoExitMessage)
   }
 
+  fn no_quiet(&self) -> bool {
+    !self.attributes.contains(&Attribute::NoQuiet)
```

Comment:
Should this be inverted, i.e.:

```suggestion
    self.attributes.contains(&Attribute::NoQuiet)
```

Reply:
done